docs/architecture/blazor-for-web-forms-developers/state-management.md

```diff
@@ -1,22 +1,98 @@
 ---
 title: State management
 description: Learn about different approaches for managing state in ASP.NET Web Forms and Blazor.
-author: danroth27
-ms.author: daroth
-ms.date: 09/11/2019
+author: csharpfritz
+ms.author: jefritz
+ms.date: 15/04/2020
 ---
 # State management
 
 [!INCLUDE [book-preview](../../../includes/book-preview.md)]
 
-*This content is coming soon.*
+State management is a key concept of Web Forms applications, facilitated through View State, Session State, Application State, and Postback features. These stateful features of the framework helped to hide the state management required for an application and allowed application developers to focus on delivering their functionality.  With ASP.NET Core and Blazor, some of these features have been relocated and some have been removed altogether.  In this chapter, we will review how to maintain state and deliver the same functionality with the new features in Blazor.
```

Comment:
All throughout, convert from 2 spaces after a sentence to 1 space.

Reply:
Converted in f952720